src/main/java/ClientGet.java

```diff
@@ -9,24 +11,29 @@
 
 public class ClientGet {
 
+
+    private static CurrenciesExchangeRate createCurrenciesExchangeRate(Currencies first, Currencies second, String output) {
+        String split[] = output.split(":");
+        String exchangeValueString = split[1].replace("}", "");
+        double exchangeValue = Double.parseDouble(exchangeValueString);
+        return new CurrenciesExchangeRate(first, second, exchangeValue);
+
+    }
+
     public static void main(String[] args) throws IOException {
 
 
         Currencies currenciesTable[] = Currencies.values();
-        List<String> currenciesExchangeRateStringList = new ArrayList<>();
         List<CurrenciesExchangeRate> currenciesExchangeRateList = new ArrayList<>();
 
 
-        for (int j = 0; j < currenciesTable.length; j++) {
-            for (int i = 0; i < currenciesTable.length; i++) {
-                CurrenciesExchangeRate currenciesExchangeRate = new CurrenciesExchangeRate();
+        for (Currencies first : currenciesTable) {
+            for (Currencies second : currenciesTable) {
 
-                currenciesExchangeRate.setFirstCurrency(currenciesTable[j].toString());
-                currenciesExchangeRate.setSecondCurrency(currenciesTable[i].toString());
 
-                if (currenciesTable[i] != currenciesTable[j]) {
+                if (second != first) {
```

Comment:
Hmm, it's better to use: if (!second.equals(first)) to indicate that you use equals method here.